tests/main/tests/integration/application-test.js

```diff
@@ -135,62 +132,3 @@ module('integration/application - Using the store as a service', function (hooks
     assert.notStrictEqual(store, secondService, 'the store can be used as a service');
   });
 });
-
-module('integration/application - Attaching initializer', function (hooks) {
```

Comment:
Are these test changes intentional?

Reply:
these tests test a removed thing, by testing the removed thing they cause a deprecation because its not really there. They were unrelated but I was tired of the noise when running the test suite